image.yaml

```diff
@@ -27,7 +27,8 @@ envs:
     - name: HTTPS_ENABLE_HTTP2
       value: "true"
     - name: "JAVA_OPTS_APPEND"
-      example: "-Dfoo=bar"
+      value: "-Dee8.preview.mode=false"
+      example: "-Dfoo=bar -Dee8.preview.mode=false"
```

Comment:
We don't want to suggest -Dee8.preview.mode=false via an example as that implies that that's a necessary setting.  Plus the 2nd comment I made for L30 applies here as well. 

I think discussion of -Dee8.preview.mode is best left to the docs.